modules/admin_manual/pages/_partials/configuration/server/occ_command/system_command.adoc

```diff
@@ -6,13 +6,66 @@
   system:cron             Execute background jobs as cron
 ----
 
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} -h system:cron
+-Usage:
+  system:cron [options]
+----
+
+== Options
+
+[width="100%",cols="20%,70%",]
+|===
+| -p, --progress        
+| Shows a progress bar - for use in manual execution. 
+Do not use when executing from crontab
+
+| -h, --help            
```

Comment:
You can remove all standard options like -h, -q ect as they are described generally on top and are valid for all commands. Only the particular options/arguments need to be described.

Reply:
Sounds fair.